src/runtimes/node/utils/entry_file.ts

```diff
@@ -8,11 +9,21 @@ export interface EntryFile {
   filename: string
 }
 
-const getEntryFileContents = (mainPath: string, moduleFormat: string) => {
+const getEntryFileContents = (mainPath: string, moduleFormat: string, layers: string[]) => {
   const importPath = `.${mainPath.startsWith('/') ? mainPath : `/${mainPath}`}`
 
   if (moduleFormat === ModuleFormat.COMMONJS) {
-    return `module.exports = require('${importPath}')`
+    if (layers.length === 0) {
```

Comment:
We could get rid of this `if` and just use the code below for everything, since `getLayersBootstrap()` will return an empty array if there are no layers. Still, in the spirit of reducing the blast radius, I want to keep the existing code path unmodified. We can have a feature flag in Netlify Build that decides whether to populate the `layers` config property.